pkgs/development/libraries/glib/default.nix

```diff
@@ -269,13 +269,7 @@ stdenv.mkDerivation (finalAttrs: {
     ln -s $PWD/glib/libglib-${librarySuffix} $out/lib/libglib-${librarySuffix}
   '';
 
-  checkPhase = ''
-    runHook preCheck
-
-    meson test --print-errorlogs
-
-    runHook postCheck
-  '';
+  mesonCheckFlags = [ "--print-errorlogs" ];
```

Comment:
Unrelated to this PR, I wonder if we should not have the setup hook do this by default. Unless one is building with `--keep-failed` the logs will be discarded, requiring building one more time when debugging.

Reply:
I was thinking that.  I'll do that next.